src/native_driver.jl

```diff
@@ -179,4 +205,69 @@ function Distributed.manage(manager::K8sClusterManager, id::Integer, config::Wor
             notify(DEREGISTER_ALERT; all=true)
         end
     end
+
+    return nothing
+end
+
+# Stripped down and modified version of:
+# https://github.com/JuliaLang/julia/blob/844c20dd63870aa5b369b85038f0523d7d79308a/stdlib/Distributed/src/managers.jl#L567-L632
+function Distributed.connect(manager::K8sClusterManager, pid::Int, config::WorkerConfig)
+    if config.connect_at !== nothing
+        # this is a worker-to-worker setup call.
+        return Distributed.connect_w2w(pid, config)
```

Comment:
hwo annoying would it be to add a test for this?  need to launch two pods from local manager?

Reply:
>  need to launch two pods from local manager?

I'm going to delete this line as currently we're never using this. That may seem strange but by default a worker ends up calling `Distributed.connect(::DefaultClusterManager, ...)` unless we overwrite [`init_worker`](https://github.com/JuliaLang/julia/blob/f7554b5c9f0f580a9fcf5c7b8b9a83b678e2f48a/stdlib/Distributed/src/cluster.jl#L372).